rust_src/src/fonts.rs

```diff
@@ -190,7 +196,12 @@ impl From<LispObject> for Option<LispFontSpecRef> {
     fn from(o: LispObject) -> Self {
         o.as_vectorlike().and_then(|v| {
             if v.is_pseudovector(pvec_type::PVEC_FONT) && o.is_font_spec() {
```

Comment:
`transmute(o)` tries to turn the LispObject directly into a pointer.
`transmute(v)` takes the Lisp_Vectorlike pointer and turns it into a LispFontSpecRef pointer.

Since we already made the V using it makes sense to me. I would lean towards @cantsin's solution.

Reply:
I prefer @cantsin's solution as well. And I can confirm that using `o` instead of `v` was a typo.